src/style.css

```diff
@@ -1,3 +1,81 @@
-body {
+:root {
+  --light-blue: #00acdc;
+}
+
+* {
   margin: 0;
+  font-family: Roboto, serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+header {
+  box-sizing: border-box;
+  padding-left: 50px;
+  padding-right: 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+}
+
+img {
+  display: block;
+}
+
+ul {
+  padding-inline-start: 0;
+  display: inline;
+}
+
+li {
+  display: inline-block;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.links {
```

Comment:
Be consistent with your vertical margins (Add only top or only bottom margin, but not both). It seems that your '.nav__link' class adds a margin-left, consider using padding on the parent element instead if spacing is needed.